artbutton.js

```diff
@@ -279,6 +304,11 @@ class ShowArt {
 	 * @memberof ShowArt
 	 */
 	static prepTokenHUD(hud, html, token) {
+		// check setting, early out if hud button is disabled
+		if(!this._cachedEnableHudButton) {
+			return;
+		}
+
```

Comment:
Prefer the single-line if syntax `if (condition) return;` for if blocks with only one statement (especially if it's `return`). Additionally, prefer spaces between `if` and the parenthetical `if (` rather than `if(`.